doc/rst/technotes/libraries.rst

```diff
@@ -384,6 +416,68 @@ For example:
    Make sure your Python functionality is also complete before calling this
    function.
 
+.. note::
+
+  If you are taking advantage of the generated ``__init__.py`` initializer
+  file to import the output directory as a package, you do not need to call
+  ``chpl_cleanup()`` yourself because it is already registered to be called
+  at program exit. The generated initializer is explained below.
+
+Python Init File
+----------------
+
+A simple ``__init__.py`` file is generated in the output directory along with
+the Python module. It looks roughly like the following:
+
+.. code-block:: python
+
+  import atexit
+
+  #
+  # Here directoryName is the name of the directory containing your Python
+  # module, and moduleName is the name of your Python module.
+  #
+  from directoryName.moduleName import *
+
+  atexit.register(moduleName.chpl_cleanup)
+
+The initializer file makes working with generated Python code easier than
+ever by transforming the output directory into a Python package. The
+initializer file will also register ``chpl_cleanup()`` to be called
+automatically at program exit.
+
+To take advantage of these new features, the directory containing the
+generated Python module needs to be visible to the script that imports it
+in some fashion, whether that be by making it a child package or by
+placing the package directory on your ``PYTHONPATH`` environment variable.
+
+.. code-block:: bash
+
+  # Builds Python module as foo/foo.py
+  chpl --library-python foo.chpl
+
+  # Adds the current directory to your PYTHONPATH
+  export PYTHONPATH="$PWD"
```

Comment:
Users may want to prepend to their python path more commonly:

```suggestion
  export PYTHONPATH="$PWD:$PYTHONPATH"
```

Reply:
Can't believe I forgot to do that. Thanks!